src/MagnumPlugins/UfbxImporter/Test/UfbxImporterTest.cpp

```diff
@@ -287,13 +336,37 @@ UfbxImporterTest::UfbxImporterTest() {
               &UfbxImporterTest::geometryCache,
               &UfbxImporterTest::geometryCacheFileCallback,
 
-              &UfbxImporterTest::staticSkin});
+              &UfbxImporterTest::staticSkin,
+
+              &UfbxImporterTest::animationInterpolation});
```

Comment:
Could you ensure the tests are added in the order they appear in the file? It makes it less confusing to follow when the tests crashes midway through, for example.

Reply:
Went far enough to do a little Python script so now everything is for sure in order 😅